Dockerfile

```diff
@@ -5,8 +5,9 @@ RUN apt-get update && \
     apt-get -y install nodejs && \
     apt-get -y clean
 
-COPY . /build-window
+RUN gem install bundler
+COPY ./Gemfile /tmp
+COPY ./Gemfile.lock /tmp
+RUN cd /tmp && bundle install
```

Comment:
I'm not sure I understand this change – why do we need to run `bundle install` in a temp directory?

Reply:
Hi! Glad to see your comments :)
I am afraid it was a bit ago I made these changes. I vaguely remember there where some dependency issues and the current docker container did not work for us but after installing dependencies with bundle install it worked. 

As I understand bundle install installs the dependencies listed in the Gemfile:
https://bundler.io/man/bundle-install.1.html

My goal was to make the container self contained without further prep/installations.

Reply:
@rouanw 
I wonder do we really need the Gemfile.lock? Isn't that a user specific file since it contains same content as Gemfile.
I read it here:

https://medium.com/@livnoorbrar/should-you-commit-gemfile-lock-or-not-9fbca418bddd

Reply:
`/tmp` or `/build-window` is an internal folder inside the docker container, so I just reused the Linux root filesystem`/tmp` folder but I can change it back. I will also remove the `Gemfile.lock` since its a local user file and not needed for installation.

Reply:
fixed removed `/tmp` folder